core/src/main/java/hudson/PluginManager.java

```diff
@@ -610,11 +610,11 @@ void considerDetachedPlugin(String shortName) {
             LOGGER.fine(() -> "not considering loading a detached dependency " + shortName + " as it is already on disk");
             return;
         }
-        LOGGER.fine(() -> "considering loading a detached dependency " + shortName);
+        LOGGER.finer(() -> "considering loading a detached dependency " + shortName);
         for (String loadedFile : loadPluginsFromWar("/WEB-INF/detached-plugins", (dir, name) -> normalisePluginName(name).equals(shortName))) {
             String loaded = normalisePluginName(loadedFile);
             File arc = new File(rootDir, loaded + ".jpi");
-            LOGGER.info(() -> "Loading a detached plugin as a dependency: " + arc);
+            LOGGER.finer(() -> "Loading a detached plugin as a dependency: " + arc);
```

Comment:
This an important debugging help and shouldn't be lower than the default visible INFO.